rest/data/project_settings.go

```diff
@@ -134,12 +134,11 @@ func SaveProjectSettingsForSection(ctx context.Context, projectId string, change
 			if err = handleGithubConflicts(mergedBeforeRef, "Changing owner/repo"); err != nil {
 				return nil, err
 			}
-			// check if webhook is enabled if the owner/repo has changed
+			// Check if webhook is enabled if the owner/repo has changed
 			_, err = model.EnableWebhooks(ctx, mergedProjectRef)
 			if err != nil {
 				return nil, errors.Wrapf(err, "enabling webhooks for project '%s'", projectId)
 			}
-			modified = true
```

Comment:
why is this no longer needed?

Reply:
I was thinking there wouldn't be anything to log here but actually I think that's inaccurate since we log if webhooks have been enabled, so putting back.